files/KoreBuild/modules/vstest/module.targets

```diff
@@ -54,4 +55,13 @@ Runs the VSTest on all projects in the ProjectsToTest itemgroup.
       ContinueOnError="$(_TestContinueOnError)"
       RemoveProperties="$(_BuildPropertiesToRemove)" />
   </Target>
+
+  <Target Name="ValidateBenchmarks" DependsOnTargets="ResolveSolutions" Condition=" '$(EnableBenchmarkValidation)' == 'true' ">
+
+    <Message Text="%0ARunning benchmark validation for:%0A@(BenchmarksToValidate -> '%(FileName)','%0A')%0A"
+      Importance="High"
+      Condition="'@(BenchmarksToValidate)' != ''" />
+
+    <Exec Command="dotnet run --no-build --project %(BenchmarksToValidate.Identity) -- * --validate-fast" IgnoreStandardErrorWarningFormat="true" />
```

Comment:
Does this require Configuration=Release?

Reply:
`JitOptimizationsValidator.FailOnError` is only added in non validation run.
https://github.com/aspnet/DependencyInjection/pull/594/files#diff-45f9a437086310818894446a2b7079fdR21